### PR TITLE
Restructure Index Page

### PR DESCRIPTION
**Pull request by hoyer-a**
Fabian and i talked about adding cards for the main sections on the index page instead of the toctree
 (e.g. _Getting Started, Documentation, SOFA Conventions, Contributing, Other_).

So here is a draft:

https://sofar--98.org.readthedocs.build/en/98/sofar.html

### CODE REVIEW

**Review by ahms5 — COMMENTED** [review]
please don't delete the toc tree, otherwise the structure of the doc is not correct. you can hide the toctree by using ``:hidden:``as well and then add the cards with the links. in this way the left sidebar should still work as before.

**Comment by hoyer-a** [comment]
Oops, yeah... fixing it right now. Thanks!

**Review by ahms5 — CHANGES_REQUESTED** [review]
thank you, looks nice!

**Comment by hoyer-a** [comment]
Works well with the fa-icons.
We could move the inline css into a class in custom.css on gallery.

**Review by ahms5 — COMMENTED** [review]
Maybe can make a draft in the css before we put it in the gallery, we could temporarily put it in this pr. Also, the logos are now very small, is that intentional? 
I think we should make it properly before we merge it. based on some of my learning during the doc gallery story

**Comment by f-brinkmann** [comment]
+1 for logos to be a little larger if possible

**Review by ahms5 — APPROVED** [review]
nice, thank you, even darkmode is working

**Review by f-brinkmann — DISMISSED** [review]
Thanks a lot, really nice. I opened #102 to take track of the css style

**Review by mberz — CHANGES_REQUESTED** [review]
I really like the new icon design elements.

Two things though:
- I'd rename the 'documentation' to something more explicit like API reference.
- The SOFA object and sofar functions sub-pages in the TOC aren't really meaningful to people who are new and not already aware of how things are structured. So I'd prefer more descriptive names in terms of functionality.

File: docs/sofar.rst [hunk 1]
```diff
@@ -2,36 +2,81 @@ sofar
...
-.. toctree::
-   :caption: Other
-   :maxdepth: 1
+    .. grid-item-card::
+        :link: documentation
+        :link-type: ref
+        :text-align: center
 
-   history
+        **Documentation**
```
[the remark this answers is not in the document]
Comment (hoyer-a) [reply docs/sofar.rst:41]: I'm fine with API Reference instead of documentation, makes more sense to me too

**Comment by ahms5** [comment]
>     * The SOFA object and sofar functions sub-pages in the TOC aren't really meaningful to people who are new and not already aware of how things are structured. So I'd prefer more descriptive names in terms of functionality.

fair point, why do we split this anyway. we could put it into on page and add a gernal introduction instead of having two intros. in other words we merge ``sofar.functions.rst`` and ``sofar.Sofa.rst`` into ``documentation.rst``

**Comment by mberz** [comment]
> > ```
> > * The SOFA object and sofar functions sub-pages in the TOC aren't really meaningful to people who are new and not already aware of how things are structured. So I'd prefer more descriptive names in terms of functionality.
> > ```
> 
> fair point, why do we split this anyway. we could put it into on page and add a gernal introduction instead of having two intros. in other words we merge `sofar.functions.rst` and `sofar.Sofa.rst` into `documentation.rst`

Sounds reasonable ;)

**Comment by f-brinkmann** [comment]
I'll do a draft for that!

**Review by f-brinkmann — DISMISSED** [review]

File: docs/_static/css/fa_icon_style_draft.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* to be moved to custom.css on gallery once approved */
+
+.icon-style {
+    color: #18609d;
+    font-size: 5.2rem;
+    margin: 1rem;
+}
```
Comment (ahms5) [inline docs/_static/css/fa_icon_style_draft.css:7]: remove

File: docs/conf.py [hunk 3]
```diff
@@ -97,13 +97,17 @@
 
 html_theme = 'pydata_sphinx_theme'
 html_static_path = ['_static']
-html_css_files = ['css/custom.css']
+html_css_files = ['css/custom.css',
+                  'css/fa_icon_style_draft.css'] # to be removed
```
Comment (ahms5) [inline docs/conf.py:101]:
```suggestion
html_css_files = ['css/custom.css']
```

**Review by ahms5 — CHANGES_REQUESTED** [review]
i have merged it into main in gallery, so we can use the css from the gallery now.
otherwise approved.
thank you all, now its really nice

**Review by f-brinkmann — APPROVED** [review]
Looks good to me, thanks

**Review by mberz — COMMENTED** [review]
Only typos. Otherwise approved. Looks good overall and is now more intuitive to comprehend.

**Review by mberz — APPROVED** [review]

**Review by ahms5 — APPROVED** [review]

**Review by f-brinkmann — APPROVED** [review]

**Pull request merged**